### PR TITLE
edits

### PR DESCRIPTION
**Pull request by shansrini**
Hi Danna,

Can you review the sections : 
Create a custom role for use with Genesys Cloud OAuth client
Import and publish the script
-Shanthi

### CODE REVIEW

File: blueprint/index.md [hunk 1]
```diff
@@ -175,25 +176,25 @@ To enable communication from Genesys Cloud to Microsoft Azure and Microsoft Team
       **Note:** Click **Yes** in the Publish Action window to publish the data action. The import action modifies only the data action configuration and not the data action contract.
       :::
 
-6.  Return to the web services data action integration and verify that the data action is in **Active** status.
+   6.  Return to the web services data action integration and verify that the data action is in **Active** status.
 
 ### Create a custom role for use with Genesys Cloud OAuth client
 
-1. Navigate to **Roles/Permissions**.
+1. Navigate to **Roles/Permissions** and click **Add Role**.
```
Comment (DannaShirley) [inline blueprint/index.md:184]: **Roles** > **Permissions**

File: blueprint/index.md [hunk 2]
```diff
@@ -175,25 +176,25 @@ To enable communication from Genesys Cloud to Microsoft Azure and Microsoft Team
...
 
    ![Create Custom Role](images/createRole.png "Create Custom Role")
 
-2. Type a **Name** for your custom role.
+2. Enter a name for your custom role.
 
-  ![Name Custom Role](images/nameCustomRole.png "Name Custom Role")
+   ![Name Custom Role](images/nameCustomRole.png "Name Custom Role")
 
-3. Search and select the **Conversation>message>Create** and **messaging>sms>send** permissions and click **Save** to assign the appropriate permissions to your custom role.
```
Comment (DannaShirley) [inline blueprint/index.md:190]: To assign the appropriate permissions to your custom role, search for and select the .... permissions. Then click **Save**.

File: blueprint/index.md [hunk 3]
```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
...
    ![Open the Script menu](images/6COpenScriptDropdown.png "Open the Script menu")
 
-5. Click the **Actions** icon and then click **Escalate to Teams**
+5. Click the **Actions** icon and then select **Escalate to Teams**.
 
-  ![Click Script Actions](images/clickScriptActions.png "Click Script Actions")
+   ![Click Script Actions](images/clickScriptActions.png "Click Script Actions")
 
-6. Expand the First Data Action
+6. Expand the first data action **Data Actions.Execute Data Action**, that creates the Teams meeting.
```
Comment (DannaShirley) [inline blueprint/index.md]: I found this a little confusing. How about, "Expand the first data action called **Data Actions. Execute Data Action**. This data action creates the Teams meeting.

File: blueprint/index.md [hunk 4]
```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
...
+11. From the Category list, select the category of the Send SMS data action.  From the Data Action list, select the Send SMS data action.
 
- ![Map Second Data Action](images/mapSecondDataAction.png "Map Second Data Action")
+    ![Map Second Data Action](images/mapSecondDataAction.png "Map Second Data Action")
 
-13. Expand the input variables for the Second Data Action
+12. Expand the input variables for the second data action.
 
-14. Type one of your purchased SMS Numbers from your Genesys Cloud organization in the **fromAddress** input variable.
+13. In the **fromAddress** input variable, type one of your SMS numbers bought from the Genesys Cloud organization.
```
Comment (DannaShirley) [inline blueprint/index.md]: type one of the SMS numbers that was purchased for your Genesys Cloud organization
(It's my understanding that they can either use SMS numbers that they purchase directly from us, or they can use SMS numbers that they purchased elsewhere but that were registered for use with us.)

File: blueprint/index.md [hunk 5]
```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
...
+6. Expand the first data action **Data Actions.Execute Data Action**, that creates the Teams meeting.
 
    ![Expand the First Data Action](images/expandFirstDataAction.png "Expand the First Data Action")
 
-7. From the **Category** drop menu, select the category of your "Create Teams Meeting" data action.  From the **Data Action** drop menu, select your "Create Teams Meeting" data action.
-
-  ![Expand the First Data Action](images/mapFirstDataAction.png "Expand the First Data Action")
-
-8. Expand the input variables for the First Data Action
+7. From the Category list, select the Teams meeting web services data action. From the Data Action list, select the Teams meeting data action.
```
Comment (DannaShirley) [inline blueprint/index.md:298]: Should the name of the data action be bolded here?

File: blueprint/index.md [hunk 6]
```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
...
+      ![Map First Data Action Input Variable](images/mapFirstDataActionDefineUserInputVariable.png "Map First Data Action Input Variable")
 
-10. Define static values for the remaining input variables.
+9. Define the static values for the input variables, `startTime`, `endTime`, and `timeZone`. 
 
-  :::primary
-  **Note:** Using the values in this example will create ISO-8601 formatted timestamps.  The startTime and endTime parameters can be the same and it can occur in the past.  If you would like to define your own static timestamps for these variables, you are welcome to do so.
-  :::
+   :::primary
+   **Note:** You can enter the same value for `startTime` and `endTime` parameters and the timestamp can be in the past. Also, define your own timestamp format. The example shows the value to create the ISO-8601 format timestamp.
```
Comment (DannaShirley) [inline blueprint/index.md]: Also, you can define your own timestamp...

File: blueprint/index.md [hunk 7]
```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
...
-  :::
+   :::primary
+   **Note:** You can enter the same value for `startTime` and `endTime` parameters and the timestamp can be in the past. Also, define your own timestamp format. The example shows the value to create the ISO-8601 format timestamp. 
+   :::
 
-  ![Define Static Input Variables](images/mapFirstDataActionDefineInputVariables.png "Define Static Input Variables")
+   ![Define Static Input Variables](images/mapFirstDataActionDefineInputVariables.png "Define Static Input Variables")
 
-11. Expand the second data action.
+10. Expand the second data action that sends the SMS.
```
Comment (DannaShirley) [inline blueprint/index.md]: Does the second data action have a predefined name? Or is it user-defined? Also, "...SMS message."

File: blueprint/index.md [hunk 8]
```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
...
 
-   ![Expand the second data action.](images/expandSecondDataAction.png "Expand the second data action.")
+    ![Expand the second data action.](images/expandSecondDataAction.png "Expand the second data action.")
 
-12. From the **Category** drop menu, select the category of your "Send SMS" data action.  From the **Data Action** drop menu, select your "Send SMS" data action.
+11. From the Category list, select the category of the Send SMS data action.  From the Data Action list, select the Send SMS data action.
```
Comment (DannaShirley) [inline blueprint/index.md]: The category and data action names should be bold.

File: blueprint/index.md [hunk 9]
```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
...
 
-14. Type one of your purchased SMS Numbers from your Genesys Cloud organization in the **fromAddress** input variable.
+13. In the **fromAddress** input variable, type one of your SMS numbers bought from the Genesys Cloud organization.
 
- :::primary
- **Note:** See additional resource below for detailed steps for purchasing an SMS number.  Phone must be typed in +11234567890 format.
- :::
+      :::primary
+      **Note:** See [Purchase SMS long code numbers](https://help.mypurecloud.com/articles/purchase-sms-long-code-numbers/ "Opens the Purchase SMS long code numbers article") in the Genesys Cloud Resource Center for detailed steps about purchasing an SMS number. You must enter the SMS number in +11234567890 format.
+      :::
```
Comment (DannaShirley) [inline blueprint/index.md:331]: For cross references, use the phrasing..."For more information about..., see LINK."

File: blueprint/index.md [hunk 10]
```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
...
+    ![Define From Address Input Variable](images/mapSecondDataActionFromAddressVariable.png "Define From Address Input Variable")
 
-15. Confirm the remaining input variables of the second data action match the example below.
+14. Enter the values for the remaining input variables of the second data action as shown in the example.
 
- ![Map Second Data Action Input Variables](images/mapSecondDataActionVariables.png "Map Second Data Action Input Variables")
+    ![Map Second Data Action Input Variables](images/mapSecondDataActionVariables.png "Map Second Data Action Input Variables")
 
-16. Below the Custom Action Name, click **Save**.
+15. Under the Custom Action Name section, click **Save**.
```
Comment (DannaShirley) [inline blueprint/index.md]: Under **Custom Action Name**, click **Save**.

File: blueprint/index.md [hunk 11]
```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
...
+    ![Save Custom Action](images/saveScriptCustomAction.png "Save Custom Action")
 
-17. In the Script menu, click **Save**.
+16. In the Script menu, click **Save**.
 
-   ![Save the script](images/saveScript.png "Save the script")
+    ![Save the script](images/saveScript.png "Save the script")
 
-18. In the Script menu, click **Publish**.
+17. In the Script menu, click **Publish**.
```
Comment (DannaShirley) [inline blueprint/index.md]: On the **Script** menu, click ...

File: blueprint/index.md (hunk 1 above)
Comment (shansrini) [reply blueprint/index.md:184]: I have used Roles/Permissions as it displays the same in the UI.

**Review by DannaShirley — APPROVED** [review]

**Pull request merged**